specs/tables.md

```diff
@@ -271,3 +271,52 @@ Columns expressions in circuit:
 - Key: `key_rlc_mult`
 - ValuePrev: `is_nonce_mod * sel1 + is_balance_mod * sel2 + is_codehash_mod * sel2 + is_storage_mod * mult_diff`
 - ValueCur: `is_nonce_mod * s_mod_node_hash_rlc + is_balance_mod * c_mod_node_hash_rlc + is_codehash_mod * c_mod_node_hash_rlc + is_storage_mod * acc_c`
+
+## `copy_table`
+
+Proved by the copy circuit.
+
+The copy table consists of 13 columns, described as follows:
+
+- **q_step**: a fixed column for boolean value to indicate a copy step, always alternating between 1 and 0, where 1 indicates a read op and 0 indicates a write op.
+- **is_first**: a boolean value to indicate the first row in a copy event.
+- **is_last**: a boolean value to indicate the last row in a copy event.
+- **ID**: could be `$txID`, `$callID`, `$codeHash` (RLC encoded).
+- **Type**: indicates the type of data source, including `Memory`, `Bytecode`, `TxCalldata`, `TxLog`.
+- **Address**: indicates the address in the source data, could be memory address, byte index in the bytecode, tx call data, and tx log data. When the data type is `TxLog`, the address is the combination of byte index, `TxLogFieldTag.Data` tag, and `LogID`.
```

Comment:
after #214 merged, `Address` for `TxLog` change to `log_id` + `field_tag` + `index` now.  maybe need update accordingly . thanks !

Reply:
Yes, I already updated the txlog lookup in the latest commit. could you help take a look?